schema-library-events/src/main/avro/connectorConfig.avdl

```diff
@@ -0,0 +1,36 @@
+@namespace("com.homeaway.digitalplatform.streamregistry")
+
+protocol ConnectorConfigProtocol {
+
+  /**
+  * S3 Connector gets data from Kafka and dumps it in S3
+  */
+  record Connector {
+
+  /**
+   * Name of the connector
+   */
+    string name;
+
+  /**
+    * Type of connector (source or sink)
+  */
+    string type;
```

Comment:
How is the type information used? Is it necessary for this PR?

If it is only source/sink, should this be an enum?

Reply:
The `type` information will be used to determine if a connector is a `source` or a `sink` .. Yes, it is necessary for this PR (although it might actually be used once other connectors are supported) .. since this information just gets passed on to other components, I chose to not harden the values that could be sent in here ..

Reply:
@krishnatg can you describe this PR in context of #33 and #34 ?